examples/with-ant-design/app/AntdRegistry.tsx

```diff
@@ -3,13 +3,11 @@
 import React from 'react'
 import { useServerInsertedHTML } from 'next/navigation'
 import { StyleProvider, createCache, extractStyle } from '@ant-design/cssinjs'
+// if you are using Nextjs 14, use below import instead. More info: https://github.com/ant-design/ant-design/issues/45567
+// import { StyleProvider, createCache, extractStyle } from '@ant-design/cssinjs/lib'
```

Comment:
Let's just update to Next 14 instead?

Reply:
Do you mean we use `import { xxx } from '@ant-design/cssinjs/lib'` directly?

Reply:
updated

Reply:
Hi, since the last problem has been solved, I changed the PR content and PR title, please review again.